common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -821,6 +821,7 @@ def _get_answer_notification(self, render_notifications):
 
             # Build the notification message based on the notification type and translate it.
             ungettext = self.runtime.service(self, "i18n").ungettext
+            gettext = self.runtime.service(self, "i18n").gettext
```

Comment:
I would prefer that we use the same pattern as the rest of the file.  This line should be:
```
_ = self.runtime.service(self, "i18n").ugettext
```
Then we don't need to make the other changes below.

Reply:
BTW, it looks like we should also add that line to get_problem_html in that file.

Reply:
Thank you @nedbat . All done